var/spack/repos/builtin/packages/neurodamus-core/package.py

```diff
@@ -36,6 +36,11 @@ class NeurodamusCore(Package):
     depends_on('py-enum34',        type=('run',), when='^python@2.4:2.7.999,3.1:3.3.999')
     depends_on('py-lazy-property', type=('run'), when='+python')
 
+    def get_commit_hash(self):
```

Comment:
Maybe no need to implement a function that is only called once and can be in a single line
`git_commit_hash = self.fetcher[0].get_commit_hash()`

Reply:
I think you are right, since they are used only once